symfony/messenger/4.3/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "#1": "MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages"
```

Comment:
Now that we have multiple good transports (doctrine and redis), I think it makes sene to give one example for each (amqp, doctrine, redis) so the user can choose